app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
```

Comment:
Check out our checklist and clean code recommendations. Use ( ) for multiple imports